src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1548,6 +1548,22 @@ int fibonacci(int n)
 }}
 ";
 
+        private static readonly string s_preferStaticAnonymousFunction = $$"""
+            using System;
+            //[
+            // {{ServicesVSResources.Prefer_colon}}
+            Func<int, int> f1 = static i => i * i;
+            Func<int, int> f2 = static delegate (int i) { return i * i; };
```

Comment:
honestly, just showing a single example is fine for these.  we dont' try to make them exhaustive.  i'd just show the lambda.